modules/sandboxed-containers-deleting-namespace-cli.adoc

```diff
@@ -0,0 +1,25 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+:_content-type: PROCEDURE
+[id="sandboxed-containers-deleting-namespace-cli_{context}"]
+= Deleting {sandboxed-containers-operator} namespace
+
+After you run the preceding commands, your cluster is restored to the state that it was prior to the installation process. You can now revoke namespace access to the operator by deleting the `openshift-sandboxed-containers-operator` namespace.
+
+.Prerequisites
+
+* You have {product-title} 4.10 installed on your cluster.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the comand-line JSON processor (`jq`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Delete the {sandboxed-containers-first} namespace by running the following command:
++
+[source, terminal]
+----
+$ oc delete namespace openshift-sandboxed-containers-operator
+----
```

Comment:
Hi Cameron. I received the CLI commands from Abhishek, and he walked me through them. I initially created a separate section for Deleting the Namespace from the CLI, to match what we have for the Web section. I decided in the end to include this command as the last command in the Deleting the Operator from the CLI section, firstly because that's how I received it from Abhishek, and also to avoid duplicating the overview information. If you think it should be separate let me know. I thought I had created the file but not added it to the assembly, but it looks like I added it to the assembly by mistake (still learning the ropes here!).